Ptt/Flows/UserInfoFlow/UserInfoVC.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  UserInfoVC.swift
+//  Ptt
+//
+//  Created by AnsonChen on 2023/4/19.
+//  Copyright © 2023 Ptt. All rights reserved.
+//
+
+import SwiftUI
+
+protocol UserInfoView: BaseView {
+
+}
+
+final class UserInfoVC: UIHostingController<UserInfoUIView>, UserInfoView {
```

Comment:
Do you have better suggestion?   
Seems like has to create this kind of controller for every view   
It is a bit ... silly 

I have tried this way
```
final class GeneralHostingController<V: View, B: BaseView>: UIHostingController<V>, B {

}
```

But it has error        
```
Inheritance from non-protocol, non-class type 'B'
```